src/content/en/fundamentals/security/credential-management/save-forms.md

```diff
@@ -0,0 +1,175 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2016-11-08 #}
+{# wf_published_on: 2016-11-08 #}
+
+# Save Credentials from Forms {: .page-title }
+
+{% include "web/_shared/contributors/agektmr.html" %}
+{% include "web/_shared/contributors/megginkearney.html" %}
+
+<div class="attempt-right">
+  <figure>
+    <video src="animations/credential-management-smaller.mov" style="max-height: 400px;" autoplay muted loop controls></video>
+    <figcaption>Save Credentials from sign-in forms</figcaption>
+  </figure>
+</div>
+
+Keep your registration and sign-in forms as simple as possible.
+
+Save credentials from sign-in forms
+so users won't have to sign in again when they return.
+
+To store user credentials from forms:
```

Comment:
Done-- I kept Update the UI short in matching sections (just as an FYI), as it looks better in TOC. Otherwise they match up for the most part.